app/routes/goal.py

```diff
@@ -0,0 +1,110 @@
+from flask import Blueprint, jsonify, request, make_response, abort
+from app import db
+from app.models.goal import Goal
+from app.models.task import Task
+import requests
+from datetime import datetime
+import os
+from app.routes.task import validate_item
+
+goal_bp = Blueprint("goal", __name__, url_prefix="/goals")
+@goal_bp.route("", methods=["POST"])
+def create_one_goal():
+    request_body = request.get_json()
+    if not "title" in request_body:
```

Comment:
```suggestion
    if "title" not in request_body:
```
small style note - having `<Object> not in ...` is the preferred syntax over `not <Object> in ...`